src/cpl.rs

```diff
@@ -166,44 +166,25 @@ impl Debug for CslStringList {
     }
 }
 
-/// Convenience shorthand for specifying an empty `CslStringList` to functions accepting
-/// `Into<CslStringList>`.
-///
-/// # Example
-///
-/// ```rust, no_run
-/// use gdal::cpl::CslStringList;
-/// fn count_opts<O: Into<CslStringList>>(opts: O) -> usize {
-///     opts.into().len()
-/// }
-///
-/// assert_eq!(count_opts(()), 0);
-/// ```
-impl From<()> for CslStringList {
```

Comment:
😢

Reply:
This isn't so bad, is it? I mean, it's `from(&[])` vs. `from(())`..

Reply:
Nah, I'm just being silly.  I was bothered by the verbosity of `src.make_valid(&CslStringList::new())` (vs. `src.make_valid(())`), but fell into being overly clever and not really common/idiomatic. I understand why you got rid of it.